### ISSUE TITLE
Upgrade to React Native 0.40.0

### ISSUE THREAD

**Issue by joncursi** [luggit/react-native-config#71]
I noticed that in `ios/ReactNativeConfig/ReactNativeConfig.h`, a header change might be needed to support RN 0.40.0:

```
#import <React/RCTBridgeModule.h>
```

**Comment by ujwal-setlur** [comment]
Will there be a release with this change?

**Comment by ujwal-setlur** [comment]
I see version 0.2.2 was released to handle react-native 0.40. I believe the version major number should have been bumped. Now all projects < RN 0.4.0 will break that have a dependency declared as:
```
"react-native-config": "^0.2.1"
```

We will now have to pin the version number to 0.2.1
```
"react-native-config": "0.2.1"
```

But really, the major number should have been bumped per facebook [guidelines](https://github.com/facebook/react-native/releases/tag/v0.40.0):

**Comment by darrenchiu** [comment]
same as @ujwal-setlur , upgrading to 0.2.2 caused my RN 0.39 project to break. It would be great if we can indicate this breaking change by major version number. +1

**Comment by GantMan** [comment]
correct, - this is a big issue with lots of these.  

unpublish is not working at the moment, so the key is to publish `0.3.0`and then rollback to a previous commit to publish `0.2.3`

I'd be happy to help @pedro if you'd like.   But I'll need npmjs access.

**Comment by pedro** [comment]
Oops! Sorry for the trouble folks, I just released 0.2.5 (compat with previous rn) and moved these changes to 0.3 per @GantMan's comment above (it's actually 0.3.1 😬  released 0.3.0 by accident but it should work too).

Will be more careful to not break this in future releases.

@GantMan you have npm access now!
